Dockerfile.gcc

```diff
@@ -0,0 +1,18 @@
+FROM gcc
```

Comment:
Specify the tag? Or do you want to test on latest version of gcc?

Reply:
Yes, I wanted to use the latest version and to be notified when an even newer version of gcc will cause any build problems in the future. 😄